src/pages/ProgramPage/ProgramContentListSection.tsx

```diff
@@ -127,110 +124,130 @@ const ProgramContentListSection: React.VFC<{
         }))
         .map(programContentSection => (
           <ProgramSectionBlock key={programContentSection.id}>
-            <ProgramSectionTitle className="mb-3">{programContentSection.title}</ProgramSectionTitle>
-            {programContentSection.contents.map(programContent =>
-              isMobile ? (
-                <MobileProgramContentItem
-                  key={programContent.id}
-                  isEnrolled={isEnrolled}
-                  onClick={() => {
-                    if (isEnrolled) {
-                      history.push(`/programs/${program.id}/contents/${programContent.id}?back=programs_${program.id}`)
-                    }
-                    if (programContent.displayMode === DisplayModeEnum.loginToTrial && !isAuthenticated) {
-                      const url = new URL(window.location.href)
-                      url.searchParams.set('position', Math.floor(layoutContent?.scrollTop || 0).toString())
-                      url.searchParams.set('programContentId', programContent.id)
-                      window.history.pushState({}, '', url.toString())
-                      setAuthModalVisible?.(true)
-                    }
-                  }}
-                >
-                  <Typography.Text className="d-flex align-items-center">
-                    <span>{programContent.title}</span>
-                  </Typography.Text>
+            {programContentSection.contents.length > 0 && (
+              <>
+                <StyledTitle>{formatMessage(productMessages.program.title.content)}</StyledTitle>
+                <Divider className="mt-1" />
```

Comment:
Moving the code into `map` will generate multiple `StyledTitle` and `Divider` if there are multiple `programContentSection`.

Reply:
Modified.
Thank you for catching my oversight